src/content/docs/apm/agents/python-agent/supported-features/python-event-loop-diagnostics.mdx

```diff
@@ -40,7 +40,7 @@ Event loop metrics will appear in the transaction **Breakdown table**:
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) APM & services > (select an app) Transactions > (click **See transaction table**) > (select a transaction):** The transaction’s **Breakdown table** flags event loops with asyncio.
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) APM & services >** (select an app) >> **Transactions** > (click **See transaction table**) > (select a transaction): The transaction’s **Breakdown table** flags event loops with asyncio.
```

Comment:
looks like there's two >> in there

Reply:
do we have a style standard for how these get capitalized? I notice we go in and out of caps in this path but in the others we keep caps throughout. i don't care much either way, but in case we have a style.